site/es/blog/new-in-devtools-106/index.md

```diff
@@ -137,46 +170,64 @@ Chromium bug: [1336604](https://crbug.com/1336604)
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hU6RmoRjFskL8P2ZAB9l.png", alt="LCP timings breakdown in the Performance Insights panel", width="800", height="523" %}
+
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hU6RmoRjFskL8P2ZAB9l.png", alt="Desglose de tiempos de LCP en el panel Perspectiva de Rendimiento", width="800", height="523" %}
 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/883542a3727a5bc1415ffee7c7bc7f7218d9e2a5 #}
 
 Chromium bug: [1351735](https://crbug.com/1351735)
 
 
 <!-- ## Auto-generate default name for recordings in the Recorder panel {: #recorder } -->
+## Generar automáticamente el nombre predeterminado para las grabaciones en el panel Grabadora {: #recorder }
+
 
 <!-- The **Recorder** panel now automatically generates a name for new recordings. -->
+El panel **Grabadora** ahora genera automáticamente un nombre para las nuevas grabaciones.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0TMJgVqyk7AeoWIR6Vee.png", alt="Default name for recordings in the Recorder panel", width="800", height="565" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0TMJgVqyk7AeoWIR6Vee.png", alt="Nombre predeterminado para las grabaciones en el panel Grabadora", width="800", height="565" %}
 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/fbf1466b00d1ff2c36fce81fde1b21f33b689a76 #}
 
 Chromium bug: [1351383](https://crbug.com/1351383)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Destacados varios {: #misc }
 
 <!-- - Previously, [Recorder extensions](/docs/devtools/recorder/reference/#extension-troubleshooting) don’t show up in the **Recorder** panel from time to time. ([1351416](https://crbug.com/1351416)) -->
+- Anteriormente, [Extensiones de grabadora](/docs/devtools/recorder/reference/#extension-troubleshooting) no aparecían en el panel de **Grabadora** de vez en cuando. ([1351416](https://crbug.com/1351416))
+
 <!-- - The **Styles** pane now displays a color picker for the [SVG `<stop>`](https://developer.mozilla.org/docs/Web/SVG/Element/stop) element’s `stop-color` property. ([1351096](https://crbug.com/1351096)) -->
+- El panel **Estilos** ahora muestra un selector de color para el `stop-color` del elemento [SVG `<stop>`](https://developer.mozilla.org/docs/Web/SVG/Element/stop) propiedad. ([1351096](https://crbug.com/1351096))
+
 <!-- - Identify script causing [layout](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) as the potential root causes for layout shifts in the **Performance Insights** panel. ([1343019](https://crbug.com/1343019)) -->
+- Identifique el script que está provocando que el [diseño](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) como las posibles causas fundamentales de los cambios de diseño en el panel **Perspectivas de rendimiento**. ([1343019](https://crbug.com/1343019))
```

Comment:
Let's tweak the wording here a bit to make this clearer + use the English name of the Performance Insights panel.
```suggestion
- Identifique el script que está provocando la ejecución de [layout](https://web.dev/avoid-large-complex-layouts-and-layout-thrashing/) como una posible causa fundamental de cambios de diseño con el panel **Performance Insights**. ([1343019](https://crbug.com/1343019))
 ```